### PR TITLE
Added enum Gender and refactored all classes where it used.

### PR DESCRIPTION
**Pull request by DenysMelnychenko**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Espress0JavaTeam/AirLine/pull/16" target="_blank">ReviewNinja

File: src/com/acme/model/User.java [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
+	private String login;
+	private String password;
+	private Gender gender;
+
+	public User(String name, String surname, String login, String password, Gender gender) {
+		this.name = name;
+		this.surname = surname;
+		this.login = login;
+		this.password = password;
+		this.setGender(gender);
```
Comment (VolodymyrBondarchuk) [inline src/com/acme/model/User.java]: this.gender = gender

File: src/com/acme/enums/Gender.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.acme.enums;
+
+public enum Gender {
+	MALE("male"),
+	FEMALY("femaly");
```
Comment (VolodymyrBondarchuk) [inline src/com/acme/enums/Gender.java]: female

File: src/com/acme/model/User.java (hunk 1 above)
Comment (DenysMelnychenko) [inline src/com/acme/model/User.java]: автозамена... Не увидел...

File: src/com/acme/enums/Gender.java (hunk 2 above)
Comment (DenysMelnychenko) [inline src/com/acme/enums/Gender.java]: Мой затуп... :)

**Pull request merged**